api/services.ts

```diff
@@ -59,40 +48,34 @@ export const postServicesList = async (
   return response.data
 }
 
-export const getAllTaxonomies = async (
-  filter = ''
-): Promise<TaxonomyTerm[]> => {
-  const response = await ServicesAxiosClient.get<TaxonomyTerm[]>(
+export const getAllNeeds = async (filter = ''): Promise<Need[]> => {
+  const response = await ServicesAxiosClient.get<Need[]>(
     `/taxonomies?filter=${filter}`
   )
   return response.data
 }
 
 // Airtable functions
 
-export const findAddress = (id: string): Promise<Address> => {
-  return ServicesAirtableClient.find<Address>('physical_addresses', id)
-}
-
-export const findService = (id: string): Promise<Service> => {
-  return ServicesAirtableClient.find<Service>('services', id)
+export const findService = (id: string): Promise<Resource> => {
+  return ServicesAirtableClient.find<Resource>('Resources', id)
```

Comment:
The strings based in here are Airtable names - switching them to match the tables in new Airtable base.

We should prob make the env variables and not hardcoded at some point but that's for another PR.